nodejs/eks/cluster.ts

```diff
@@ -620,6 +622,28 @@ export function createCore(
         },
     );
 
+    // apply the tags to the cluster security group if it was created by EKS
+    if (!eksClusterSecurityGroup) {
+        const eksCreatedSgId = eksCluster.vpcConfig.clusterSecurityGroupId;
+        eksClusterSecurityGroup = aws.ec2.SecurityGroup.get(
+            `${name}-eksClusterSecurityGroup`,
+            eksCreatedSgId,
+            {},
+            { parent, provider },
+        );
+        pulumi.output(args.clusterTags).apply((tags) => {
```

Comment:
This is a new feature that I was not expecting to find must have missed something from the PR description. It looks important and a nice feature.

Reply:
Had to remove now that we're not able to re-use the cluster sg